base_contract/views/agreement.xml

```diff
@@ -12,33 +12,41 @@
     <field name="model">agreement</field>
     <field name="arch" type="xml">
         <form string="Agreement">
+            <sheet>
             <div class="oe_button_box" name="button_box">
-                <button name="toggle_active" type="object"
-                    class="oe_stat_button" icon="fa-archive">
-                    <field name="active" widget="boolean_button"
-                        options='{"terminology": "archive"}'/>
-                </button>
+            </div>
```

Comment:
can you keep active button

Reply:
In v13, the active button is replaced by "Action > Archive". You can have a look at the partner form or the product form: there is not Active button any more, you must use the Action menu.

Reply:
ah, thanks for the explanation, didn't see that